packages/web-api/src/methods.ts

```diff
@@ -1407,4 +1413,27 @@ export interface ViewsUpdateArguments extends WebAPICallOptions, TokenOverridabl
   hash?: string;
 }
 
+  /*
+   * `workflows.*`
+   */
+export interface WorkflowsStepCompletedArguments extends WebAPICallOptions, TokenOverridable {
+  workflow_step_execute_id: string;
+  outputs?: object;
```

Comment:
This is user defined, so I think object type is fine for now for the beta but we might want to get more specific in the future if we can